### DISCUSSION TITLE
이미지 크기 기본 설정 추가 건의

### DISCUSSION THREAD

**Discussion by baeksh0420** [pronist/hELLO#102] [Ideas]
**환경 (버그가 발생한 환경을 적어주세요.):**
---
- 운영체제: macos
- 브라우저: chrome
- 스킨 버전: 3.6.1

**설명**
---
추가 건의드립니다!
포스팅 시 업로드 된 **이미지의 기본 너비가 글 너비와 같도록** 설정할 수 있을까요?

**발생 과정**
---
- 현재 예시
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/64413742/166906841-70c1deaf-7200-4ca6-8926-1b9881f47453.png">

**스크린샷 및 링크**
---
글과 이미지의 너비를 같게 하기 위해선 아래 두가지 방법이 존재하는 것 같습니다.

**1. 이미지 크기를 줄여서 업로드**
- 방법
![image](https://user-images.githubusercontent.com/64413742/166909727-04aa52a5-84d0-4291-ad7c-e83c3d9308d2.png)

- 결과
![image](https://user-images.githubusercontent.com/64413742/166909915-c17f06a4-1420-40fb-b9f3-3825e99f608a.png)

**2. 기본 글 너비를 넓게 설정하기**
- 방법
<img width="572" alt="image" src="https://user-images.githubusercontent.com/64413742/166908177-f25bcf39-200d-4689-94f8-912f3333bbd9.png">

- 결과
![image](https://user-images.githubusercontent.com/64413742/166910051-189842f2-580e-4744-860a-b3ecf02516ce.png)

현재도 해결 방법은 있지만, 혹시 추가 개발 계획이 있으시다면 개선 시 많은 도움이 될 것 같아 건의드립니다!
무엇보다 너무 좋은 스킨 감사합니다 🙇‍♀️

**Comment by pronist** [comment]
@baeksh0420 이미지가 본문 넓이를 벗어나는 것은 **가운데 정렬** 만 해당하므로 **본문 폭맞춤** 과 같은 형태에서는 적용되지 않습니다. 다만, 조금 더 나은 패치 방향이 있다면 가급적 On/Off 형태로 관련 기능이 필요하지 않다면 끄는 것도 제공하는 건 괜찮을지도 모르겠군요.

**Reply by baeksh0420** [reply]
앗 제가 해당 기능이 있는지 모르고 말씀드렸네요! 빠른 답변과 내용 고려 감사드립니다!!
